Java/JDI/jdi-cucumber-test-generator/src/main/java/com/epam/test_generator/entities/StepType.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.test_generator.entities;
+
+public enum StepType {
+
```

Comment:
The same thing can be done like this:
```
public enum StepType {
 private String typeName;
 public StepType(String typeName){
  this.typeName = typeName;
}
 GIVEN("Given")
...
}
```
Doing this way it's less code to write :)